src/pages/AnalyticsPage/Chart/useCoverage.js

```diff
@@ -28,14 +28,14 @@ export const useCoverage = ({ params, options = {} }) => {
     isPublic: shouldDisplayPublicReposOnly,
     opts: {
       select: (data) => {
-        if (data?.measurements?.[0]?.max === null) {
-          data.measurements[0].max = 0
+        if (data?.measurements?.[0]?.avg === null) {
+          data.measurements[0].avg = 0
         }
 
         // set prevPercent so we can reuse value if next value is null
         let prevPercent = data?.measurements?.[0]
         const coverage = data?.measurements?.map((measurement) => {
-          let coverage = measurement?.max ?? prevPercent
+          let coverage = measurement?.avg ?? prevPercent
 
           // can save on a few reassignments
```

Comment:
Curious: is this actually significant? I've never though to optimize at this level before

Reply:
It could potentially be significant. I'm not really sure how the API handles this, but I believe when we set to AVG the values we get back is the average of all the selected repos coverage, instead of just the max coverage value.

Reply:
You can do a deeper dive on the logic here, and maybe explain it to @aj-codecov and myself hah if you do

https://github.com/codecov/codecov-api/blob/main/timeseries/helpers.py#L61-L88